main-app/app/scripts/player-select/player-service/player-service.js

```diff
@@ -0,0 +1,27 @@
+angular.module('Server.MyModule')
+    .service('PlayerService',  function () {
+        var me= this;
+        me.playerArray = ["human", "random", "pre-trained"];
+        me.player1 = me.playerArray[0];
+        me.player2 = me.playerArray[0];
+        var selectCharacter = function (playerChoice) {
```

Comment:
It is possible to do this with fewer lines of code - see what you can come up with.